Breaking-Changes.md

```diff
@@ -70,9 +71,58 @@ switch(arr.length) {
 
 ```ts
 let a = 0;
-let y = (void a, 1); // no warning for `a` 
+let y = (void a, 1); // no warning for `a`
 ```
 
+## Extending built-ins like `Error`, `Array`, and `Map` may no longer work
+
+As part of substituting the value of `this` with the value returned by a `super(...)` call, subclassing `Error`, `Array`, and others may no longer work as expected.
+This is due to the fact that constructor functions for `Error`, `Array`, and the like use ES6's `new.target` to adjust the prototype chain;
+however, there is no way to emulate the semantics of `new.target` in ECMAScript 5.
```

Comment:
it might be worth noting that it was not working correctelly in TS 2.0 either for the same reason.